tests/unittests/Foundation/ArchivalTests.mm

```diff
@@ -14,15 +14,15 @@
 //
 //******************************************************************************
 
-#include <Windows.h>
+#include <windows.h>
 #include <TestFramework.h>
 #import <Foundation/Foundation.h>
 
 @interface TestCreationSignallingClass : NSObject <NSSecureCoding>
 + (void)resetCreationCount;
 @end
 
-static unsigned long creationCount;
+static long creationCount;
```

Comment:
it didn't

---
In reply to: [72313000](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72313000) [](ancestors = 72313000)